server/usecases/account/user.go

```diff
@@ -0,0 +1,94 @@
+package account
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/go-github/github"
+	domain "github.com/h3poteto/fascia/server/domains/user"
+	repo "github.com/h3poteto/fascia/server/infrastructures/user"
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+)
+
+// InjectUserRepository inject db connection and return repository instance.
+func InjectUserRepository() domain.Repository {
+	return repo.New(InjectDB())
+}
+
+// RegistrationUser registers a user.
+func RegistrationUser(email, password, passwordConfirm string) (*domain.User, error) {
+	return domain.Registration(email, password, passwordConfirm, InjectUserRepository())
+}
+
+// FindUser finds a user.
+func FindUser(id int64) (*domain.User, error) {
+	return domain.Find(id, InjectUserRepository())
+}
+
+// FindUser finds a user.
```

Comment:
__[Golint]__ comment on exported function FindUserByEmail should be of the form "FindUserByEmail ..."

<sub>posted by Sider</sub>